src/Symfony/Component/Notifier/Bridge/Telegram/TelegramOptions.php

```diff
@@ -156,4 +166,156 @@ public function answerCallbackQuery(string $callbackQueryId, bool $showAlert = f
 
         return $this;
     }
+
+    /**
+     * @return $this
+     */
+    public function location(float $latitude, float $longitude): static
+    {
+        $this->options['location'] = ['latitude' => $latitude, 'longitude' => $longitude];
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function venue(float $latitude, float $longitude, string $title, string $address): static
+    {
+        $this->options['venue'] = [
+            'latitude' => $latitude,
+            'longitude' => $longitude,
+            'title' => $title,
+            'address' => $address
+        ];
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function document(string $url): static
+    {
+        $this->options['document'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadDocument(string $path): static
+    {
+        $this->options['upload']['document'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function video(string $url): static
+    {
+        $this->options['video'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadVideo(string $path): static
+    {
+        $this->options['upload']['video'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function audio(string $url): static
+    {
+        $this->options['audio'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadAudio(string $path): static
+    {
+        $this->options['upload']['audio'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function animation(string $url): static
+    {
+        $this->options['animation'] = $url;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadAnimation(string $path): static
+    {
+        $this->options['upload']['animation'] = $path;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function sticker(string $url, string $emoji = null): static
+    {
+        $this->options['sticker'] = $url;
+        if ($emoji !== null) {
+            $this->options['emoji'] = $emoji;
+        }
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function uploadSticker(string $path, string $emoji = null): static
+    {
+        $this->options['upload']['sticker'] = $path;
+        if ($emoji !== null) {
+            $this->options['emoji'] = $emoji;
```

Comment:
shouldn't we also assign `null` here, to properly account for a case where the caller wants to change a previous config ?

Reply:
OK, based on the fact that calling several of those methods will do wonky stuff anyway, this might be fine...